src/test/system_tests/test_md_aggregator.js

```diff
@@ -273,10 +273,7 @@ function wait_for_server_to_start(max_seconds_to_wait, port) {
                 return isNotListening;
             },
             function() {
-                return P.ninvoke(request, 'get', {
-                        url: 'http://localhost:' + port,
-                        rejectUnauthorized: false,
-                    })
+                return http_utils.get('http://localhost:' + port)
```

Comment:
not http_get?
```suggestion
                return http_utils.http_get('http://localhost:' + port)
```
maybe we should also pass the options?

Reply:
url is passed as the first param, and `rejectUnauthorized` is irrelevant since it's HTTP and not HTTPS

Reply:
you are right regarding `http_get`. good catch :)